app/assets/javascripts/actions/conference_actions.es6.jsx

```diff
@@ -86,6 +74,14 @@
       };
     }
 
+    storeListAttribute(key, value, index) {
```

Comment:
You right you right. Another thing we could do is just have `primary` and `secondary` attributes in the `template`, and then on `createGroup` manually reform the params appropriately? This is an interesting case we don't see elsewhere.